src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/AssemblyContigAlignmentSignatureClassifier.java

```diff
@@ -0,0 +1,171 @@
+        return one.referenceSpan.contains(two.referenceSpan)
+                ||
+                two.referenceSpan.contains(one.referenceSpan);
+    }
+
+    /**
+     * @param contigWithOnlyOneConfigAnd2Aln assumed to have only two alignments
+     */
+    @VisibleForTesting
+    static boolean isSameChromosomeMapping(final AlignedContig contigWithOnlyOneConfigAnd2Aln) {
+        Utils.validateArg(!hasIncompletePictureDueToRefSpanContainment(contigWithOnlyOneConfigAnd2Aln),
+                "assumption that input contig has alignments whose ref span is not completely covered by the other's is violated \n" +
+                        contigWithOnlyOneConfigAnd2Aln.toString());
+        return contigWithOnlyOneConfigAnd2Aln.alignmentIntervals.get(0).referenceSpan.getContig()
+                .equals(contigWithOnlyOneConfigAnd2Aln.alignmentIntervals.get(1).referenceSpan.getContig());
+    }
+
+    /**
+     * @param contigWithOnlyOneConfigAnd2AlnToSameChr assumed to have only two alignments mapped to the same chromosome
+     */
+    @VisibleForTesting
+    static boolean indicatesIntraChrStrandSwitchBkpts(final AlignedContig contigWithOnlyOneConfigAnd2AlnToSameChr) {
+        Utils.validateArg(isSameChromosomeMapping(contigWithOnlyOneConfigAnd2AlnToSameChr),
+                "assumption that input contig's 2 alignments map to the same chr is violated. \n" +
+                        contigWithOnlyOneConfigAnd2AlnToSameChr.toString());
+        return contigWithOnlyOneConfigAnd2AlnToSameChr.alignmentIntervals.get(0).forwardStrand
+                ^
+                contigWithOnlyOneConfigAnd2AlnToSameChr.alignmentIntervals.get(1).forwardStrand;
+    }
+
+    /**
+     * @param contigWithOnlyOneConfigAnd2AlnToSameChrWithoutStrandSwitch assumed to have only two alignments mapped to the same chromosome and the same strand
+     */
+    @VisibleForTesting
+    static boolean indicatesIntraChrTandemDupBkpts(final AlignedContig contigWithOnlyOneConfigAnd2AlnToSameChrWithoutStrandSwitch) {
+        Utils.validateArg( !indicatesIntraChrStrandSwitchBkpts(contigWithOnlyOneConfigAnd2AlnToSameChrWithoutStrandSwitch),
+                "assumption that input contig's 2 alignments map to the same chr is violated. \n" +
+                        contigWithOnlyOneConfigAnd2AlnToSameChrWithoutStrandSwitch.toString());
+
+        final AlignmentInterval one = contigWithOnlyOneConfigAnd2AlnToSameChrWithoutStrandSwitch.alignmentIntervals.get(0),
+                                two = contigWithOnlyOneConfigAnd2AlnToSameChrWithoutStrandSwitch.alignmentIntervals.get(1);
+        final ChimericAlignment tempCA = new ChimericAlignment(one, two, dummyInsertionMappingStringList,
+                "dummy", null); // passing null because alignments are guanranteed to be same-chr mapping
+        return !tempCA.isNotSimpleTranslocation();
+    }
+
+    private static final List<String> dummyInsertionMappingStringList = Collections.emptyList();
```

Comment:
I don't think you need this as a separate static variable, I'd just inline it.

Reply:
done